### PR TITLE
Adding Forms with Formik and Yup

### PR DESCRIPTION
**Pull request by cfrancisco**
Besides that, was enhanced the current code following the suggestions made in code review.

### CODE REVIEW

File: src/Components/Avatar/utils.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    reader.readAsDataURL(img);
+}
+
+function beforeUpload(file) {
+    const { type, size } = file;
+    const isJpgOrPng = type === 'image/jpeg' || type === 'image/png';
+    if (!isJpgOrPng) {
+        message.error('You can only upload JPG/PNG file!');
+    }
+    const isLt2M = size / 1024 / 1024 < 2;
```
Comment (nunesjoab) [inline src/Components/Avatar/utils.js]: Shouldn't this isLt2M constant be a bool function?

File: src/Components/Button/index.test.js [hunk 2]
```diff
@@ -2,25 +2,26 @@ import React from 'react';
 import { configure, mount } from 'enzyme';
```
Comment (nunesjoab) [inline src/Components/Button/index.test.js]: Really good this test. Pretty coverage from frontend view but this file must be under \_\_tests\_\_ folder

File: src/Pages/Forms/Forms.js [hunk 3]
```diff
@@ -1,6 +1,35 @@
...
+import { Formik, Field, Form } from 'formik';
+import { TextField } from 'formik-material-ui';
+import * as Yup from 'yup';
+import { DojotButton } from '../../Components/Button';
+
+//
+// More info: https://jaredpalmer.com/formik/docs/guides/validation
+//
+
+const ranges = [
```
Comment (nunesjoab) [inline src/Pages/Forms/Forms.js:14]: Could this ranges constant be inside an utils file like Avatar component?

File: src/Components/Topbar/Topbar.js [hunk 4]
```diff
@@ -1,14 +1,18 @@
...
+import AccountCircle from '@material-ui/icons/AccountCircle';
 import clsx from 'clsx';
 import PropTypes from 'prop-types';
-
-const drawerWidth = 240;
+import Typography from '@material-ui/core/Typography';
+import configs from '../../settings';
 
 const useStyles = makeStyles((theme) => ({
     appBar: {
```
Comment (nunesjoab) [inline src/Components/Topbar/Topbar.js:18]: We need to decide what pattern use. In Button component we have a _styles.js file and inside the Button.js we import the styles and use them as parameter to makeStyles function. I think that way is better, separating style object from Component JS file.

File: src/Components/Avatar/utils.js (hunk 1 above)
Comment (cfrancisco) [reply src/Components/Avatar/utils.js]: Could be, I'll change it.

**Review by nunesjoab — APPROVED** [review]

**Pull request merged**